catkin_tools/config.py

```diff
@@ -35,6 +35,7 @@
 install: config --install
 p: create pkg
 test: build --verbose --make-args test --
+roslint: build --verbose --make-args roslint --
```

Comment:
> I hope run_tests also aliases with -iv option..

I guess just `--verbose`. It's the line below where you added the roslint alias ;-).